config/settings.yml

```diff
@@ -17,6 +17,13 @@
     :password:
     :port:
     :user:
+  :gce:
```

Comment:
I wasn't sure what to do with this... The core settings have gce, but these settings have google.  I don't think we should have both, but I don't know which is the right one.  @agrare WDYT?

Reply:
Proxy settings use the ems_type so `:gce` is correct here

Reply:
Then do we need a data migration for the exist google one just above?

Reply:
That doesn't look like it was ever used, `http_proxy_uri` uses the ems_type https://github.com/ManageIQ/manageiq/blob/master/app/models/manageiq/providers/base_manager.rb#L45-L47 which is "gce" https://github.com/ManageIQ/manageiq-providers-google/blob/master/app/models/manageiq/providers/google/cloud_manager.rb#L41